tests/functional.rs

```diff
@@ -222,6 +223,56 @@ mod divbufshared {
         assert_eq!(dbs.capacity(), cap);
         assert_eq!(dbs.len(), cap);
     }
+
+    #[test]
+    pub fn to_vec() {
+        let v = vec![1, 2, 3, 4];
+        let dbs = DivBufShared::from(v);
+
+        {
+            let mut dbm = dbs.try_mut().unwrap();
+            assert_eq!(dbm, [1, 2, 3, 4][..]);
+            dbm[0] = 5;
+            assert_eq!(dbm, [5, 2, 3, 4][..]);
+        }
+
+        let v2: Vec<u8> = dbs.try_into().unwrap();
+        assert_eq!(v2, vec![5, 2, 3, 4]);
+    }
+
+    #[test]
+    pub fn to_vec_error() {
```

Comment:
This test really tests three different, totally independent things.  Can you please split it up into three separate tests named to_vec_after_try_mut, to_vec_after_try_const, and to_vec_after_clone_inaccessible ?

Reply:
Done.